### ISSUE TITLE
em_conf.php and composer.json have opposing settings

### ISSUE THREAD

**Issue by subnoodle** [a-r-m-i-n/dce#44]
**Versions**
- DCE version: 2.9.3
- TYPO3 version: 11.5.19
- PHP version: 8.1.12

**Describe the bug**
composer.json says:
```
   "require": {
        "php": ">=7.3",

```
em_conf.php says:
```
    'constraints' => [
        'depends' => [
            'php' => '7.3.0-7.4.99',

```
Both together do not make sense.
It leads to have composer running on PHP 8.1 correctly, what is basically ok. But should faild for the em_conf-contraint and for bug #43 .
Our experience is, that DCE runs well on PHP 8 except bug #43 
So, please adapt em_conf:
```
    'constraints' => [
        'depends' => [
            'php' => '7.3.0-8.1.99',

```

or even to php 8.2.

Many thanks,
Thomas

**Comment by a-r-m-i-n** [comment]
Thanks! Composer is the correct one. DCE should work properly with 8.1... I'm also going to check the other issues.

**Comment by a-r-m-i-n** [comment]
Fixed in #62
